mm2src/coins/rpc_command/init_create_account.rs

```diff
@@ -238,7 +238,7 @@ impl RpcTask for InitCreateAccountTask {
                 MmCoinEnum::QtumCoin(qtum) => qtum.revert_creating_account(account_id).await,
                 _ => (),
             }
-        }
+        };
```

Comment:
I find the semicolons at the `if { };` and `match { };` statements unnecessary.
They make sense if only you want to get a result of it:
```rust
let res = if cond {
  10
} else {
  11
};
```
In other cases, I think it's worth not to add the semicolons. It's not critical, but please don't put them at the future PRs if it's not required

Reply:
copy that

Reply:
Yes but I also added `todo!()` and comment that `Copied` could be handled somehow.
```rust
    match platform_coin_tokens.entry(coin.ticker().to_string()) {
      Entry::Vacant(entry) => entry.insert(Vec::new()),
      Entry::Occupied(_) => todo!() // probably we should remove the coin from `CoinsContext::coins` container AND return an error, but on the other hand we could just don't insert it.
    }
```
https://github.com/KomodoPlatform/atomicDEX-API/pull/1525#discussion_r1035342604

Since you decided not to handle this case, it can be simplified :)